src/Component/Calculator.js

```diff
@@ -1,52 +1,49 @@
 /* eslint-disable react/prop-types, react/prefer-stateless-function */
```

Comment:
- [ ] Please don't disable eslint rules because you have it to help you to write
clean codes like for this prop-types case you can fix it